profiles/badbatunde.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/105541302/184447490-f249bc52-23fa-4249-abed-8d8d2b97392b.png",
+  "issueId": 1300,
+  "name": "HBO",
```

but consider entering your full name.
```suggestion
  "name": "HBO",
```

---

Thanks, I have made the modification.